index.html

```diff
@@ -266,7 +266,10 @@ <h3>
             Other Deliverables
           </h3>
           <p>
-            Other non-normative documents may be created such as:
+Other non-normative documents may be created as time, attention, and resources permit. The following list is a
+non-exhaustive selection of documents the Working Group may wish to produce as WG Notes or other documents. The Working
+Group will use its discretion to decide which, if any, to work on, and may publish WG Notes or other documents not
```

Comment:
probably paraphrase
> Group will use its discretion to decide which, if any, to work on the documents not listed here.

Reply:
made a suggestion in a line above per a comment during VCWG call on 2022-Feb-16th.